src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -781,7 +781,6 @@ void NotationViewInputController::mouseMoveEvent(QMouseEvent* event)
     }
 
     m_view->hideContextMenu();
-    m_view->hideElementPopup();
```

Comment:
This removal might have unintended side effects on other popups. Perhaps, instead of removing this line, we should make it conditional. Something like "if (!viewInteraction()->isTextEditingStarted())`